src/backend/joanie/core/models/courses.py

```diff
@@ -164,13 +164,83 @@ class Organization(parler_models.TranslatableModel, BaseModel):
     )
     signature = models.ImageField(_("signature"), blank=True)
     logo = ThumbnailerImageField(_("logo"), blank=True)
-    country = CountryField(_("country"), default=settings.JOANIE_DEFAULT_COUNTRY_CODE)
+    country = CountryField(
+        _("country"),
+        help_text=_(
+            "Country field will be deprecated soon in order to be replaced by address relation."
+        ),
+        default=settings.JOANIE_DEFAULT_COUNTRY_CODE,
+    )
+    enterprise_code = models.CharField(
+        verbose_name=_("Enterprise code"),  # e.g : SIRET in France
+        max_length=50,
+        blank=True,
+        null=True,
+    )
+    activity_category_code = models.CharField(
+        verbose_name=_("Activity category code"),  # e.g : APE in France
+        max_length=50,
+        blank=True,
+    )
+    representative_profession = models.CharField(
+        verbose_name=_("Profession of the organization's representative"),
+        help_text=_("representative profession"),
+        max_length=100,
+        blank=True,
+    )
+    # This field is optional 'signatory_representative'. If it is set, you must set the
+    # field 'signatory_representative_profession'
```

Comment:
This comment should be the text of `help_text`

Reply:
done :heavy_check_mark: